handlers/feedback.go

```diff
@@ -94,18 +99,47 @@ func addFeedback(w http.ResponseWriter, req *http.Request, rend interfaces.Rende
 		return
 	}
 
-	if ff.URL == "" {
-		ff.URL = mapper.WholeSite
-	}
+	if cfg.EnableFeedbackAPI {
+		healthClient := health.NewClient("api-router", cfg.APIRouterURL)
+		feedbackAPIClient := feedbackAPI.NewWithHealthClient(healthClient)
```

Comment:
Ideally, again, we'd again be passing in a client rather than instantiating one here. You can see how this is done in the dp-frontend-search-controller as an example. It makes it easier to mock out the client for testing and reusing client functionality across the application rather than re-doing it all the time.